include/GafferImage/Merge.h

```diff
@@ -79,7 +79,9 @@ class Merge : public ImageProcessor
 			Over,      // A + B(1-a)
 			Subtract,  // A - B
 			Difference,// fabs( A - B )
-			Under      // A(1-b) + B
+			Under,     // A(1-b) + B
+			Min,       // min( a, b )
```

Comment:
I think the lower case a/b here refers to the alpha channel doesn't it? Shouldn't it be `min( A, B )`, to refer to the actual channels being composited?